docs/examples.md

```
@@ -5,7 +5,7 @@ A collection of practical examples and code recipes for using Pixelpop in real-w
 ## 🎯 Table of Contents
 
 - [CLI Tools & Utilities](#cli-tools--utilities)
-- [Web Applications](#web-applications) 
+- [Web Applications](#web-applications)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)